machine.go

```diff
@@ -535,6 +535,12 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 
 	os.Setenv("PATH", os.Getenv("PATH") + ":/sbin:/usr/sbin")
 
+	for _, v := range m.mounts {
+		if _, err := os.Stat(v.hostDirectory); err != nil {
```

Comment:
some thoughts:
- does this stop if the volume `hostDirectory`
- does `machineDirectory` need to exist ?
- check if `machineDirectory` contains spaces
- check if `label` contains spaces
- add a unit test to check this behaviour